### DISCUSSION TITLE
WLAN HF-Leistung

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#1227] [Q&A]
Hallo,

ich habe noch einmal Fragen: 

Bei der WLAN-Verbindung zwischen openDTU und FritzBox kann man unter "WLAN-Informationen (Station)" die Parameter "Qualität" und "RSSI" ablesen. Diese Werte sind bei mir sehr gut, da die Geräte nur einige Meter Abstand haben.

Völlig anders ist die Situation zwischen openDTU und HM-800.
Hier ist eine erhebliche Entfernung zwischen openDTU im Treppenhaus im Kellerbereich bis zum WR im 1. OG zu überbrücken.
Bisher läuft die Verbindung wohl problemlos.

Zu dieser Verbindung finde ich keine Qualitäts-Werte in der openDTU.

Weiß jemand wo man die finden könnte oder ist die im System überhaupt nicht verfügbar?
Das würde mich schon sehr interessiern.

Letzte Frage noch:
Auf welche Verbindung bezieht sich die  "NRF-24 Sendeleistung".

Danke im Voraus.- und viele Grüße.

**Comment by Fokker-1** [comment]
... noch eine Ergänzung:

In der Konsole finde ich einen Wert von -80 dBM. Der steht wie festgenagelt ohne jede Änderung.
Den Code verstehe ich nicht, habe also keine Ahnung, was dieser Wert bedeuten könnte.

Danke und viele Grüße

**Reply by tbnobody** [reply]
Das NRF Modul kann nur 2 Stati für die Verbindungsqualität liefern >-64dbm und <-64dbm. Da die qualität aber intern als zahlenwert gespeichert wird kann da kein > oder < dargestellt werden. entsprechend gibt es einfach -30 und -80...

**Comment by Fokker-1** [comment]
Hallo,

super, vielen Dank für die schnelle Antwort, das ist mir jetzt klar.

Dann bliebe noch meine o.g. Frage:

"Auf welche Verbindung bezieht sich die "NRF-24 Sendeleistung?".

Vielleicht weiß jemand dazu ja auch noch eine Antwort.

Schon mal vielen Dank und Grüße.

**Reply by Solarteur** [reply]
Das NRF24-Modul ermöglicht die Verbindung zu dem/den Wechselrichter(n) - darauf bezieht sich auch die Sendeleistung. Eigene Erfahrungen zeigen, dass die Stabiltität dieser Verbindung mit höherer Sendeleistung nicht unbedingt besser wird.

**Reply by Dekalus** [reply]
Ist das NRF24-Modul nicht für beide Schnittstellen gedacht? Also vom Router (WLAN) zur OpenDTU und von der OpenDTU zum Wechselrichter?

Wenn nicht, welches Modul ist denn da für die WLAN-Kommunikation zuständig?

Wie kann ich denn nun nach dem was hier gesagt wurde, einen genauen Test über die Verbindungsqualität zwischen OpenDTU und Wechselrichter machen? Oder präziser ausgedrückt, wie kann ich feststellen, ob eine Änderung der Sendeleistung bei mir positive oder negative Auswirkungen auf die Verbindungsqualität hat?

**Reply by Solarteur** [reply]
Das ESP32-Modul ist für die WLAN-Kommunikation (Router) zuständig.

Einen **genauen** Test über die Verbindungsqualität zwischen OpenDTU und Wechselrichter gibt es nicht, s. Beitrag von tbnobody weiter oben.

Bei mir wurden in der Einstellung "Minimum" die Datenpakete störungsfrei empfangen (s. Konsole), also habe ich es so belassen.

**Reply by Dekalus** [reply]
Danke für die schnelle Antwort.

Es ist sehr schade, dass keine genaueren Analysemethoden existieren...

Wie kann ich feststellen, ob bei mir die Daten störungsfrei übermittelt wurden?

Deutet eine störungsfreie Übermittlung der Daten darauf hin, dass beim ersten Versuch ohne Laufzeitverlüste die Daten übermittelt werden konnten? Oder geht es da nur um den letztgültigen Zustand nach mehreren erfolglosen Versuchen?

**Comment by Fokker-1** [comment]
Hallo,

danke für die Infos zum NRF24-Modul.

Grüße

**Comment by Fokker-1** [comment]
@ Dekalus

Ja, genau das war der eigentliche Hintergrund meiner o. g. Frage.
Ist mir hinsichtlich der beidseitigen (!) Verbindung auch noch nicht klar.
Grüße

**Comment by Fokker-1** [comment]
Hallo,

eine weitere Frage noch:

Die openDTU liefert ja bei den Live-Daten den "Gesamtertrag insgesamt".
Gibt es dazu auch ein Datum, das irgendwo im System hinterlegt ist?

Wenn Ja, könnte man das in der Überschrift dann noch ergänzen?

Danke im Voraus und Grüße
